app/controllers/users_controller.rb

```diff
@@ -0,0 +1,32 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+      @user = User.new(user_params)
+      if @user.save
```

Comment:
after saving user, I would recommend using log_in method to sign in user after user sign up

Reply:
done...thanks